RWS-Submission_Guidelines.md

```diff
@@ -83,21 +83,18 @@ It is important that users' interests are protected from invalid submissions, an
+- Note that RWS uses the qualification of "eTLD+1 with a secure transport scheme" to determine the boundaries of an entry, meaning that `https://example.com` and `https://a.example.com` should not both exist on `related_website_sets.JSON` as they share the same eTLD+1: `example.com`. However, this should not be an issue for the owner of `https://example.com`, as its and `https://a.example.com`'s cookies are not considered third-party to each other. Cookies are bounded by site, and since `https://a.example.com` is a subdomain of `https://example.com`, both origins have access to the same cookies (modulo the `Domain` attribute) since they are the same site. To learn more about what is considered cross-site vs same-site, [please read this article](https://web.dev/articles/same-site-same-origin).
+- Each domain must not already be present in the <a href="https://github.com/googlechrome/first-party-sets/blob/main/related_website_sets.JSON">canonical RWS list.</a>
+- Each domain must satisfy the `/.well-known/` metadata requirement:
+     - The /.well-known/ metadata requirement demonstrates that the submitter has administrative access to the domains present in the set, since administrative access is required to modify the /.well-known/ file. This will help prevent unauthorized actors from adding domains to a set. 
+    - The primary domain must serve a JSON file at `/.well-known/related-website-set.json` (Note: list entries merged before September 15th 2023 may serve their well-known file at `/.well-known/first-party-set.json` instead; however, any changes to those entries will require that the primary and all members of the set must be served at `/.well-known/related-website-set.json` like any other entry). The contents of the file must be identical to the submission. Each member domain must serve a JSON file at /.well-known/related-website-set.json. The contents of the file must name the primary domain. These files must be maintained for the duration of the domain’s inclusion in the set.
+     -  Any changes to an existing RWS in the <a href="https://github.com/googlechrome/first-party-sets/blob/main/related_website_sets.JSON">canonical RWS list</a>  must also be reflected in that set's JSON files at `/.well-known/related-website-set.json`.
+     - If an RWS owner wishes to remove a set entirely from the <a href="https://github.com/googlechrome/first-party-sets/blob/main/related_website_sets.JSON">canonical RWS list,</a> then that set's primary must serve a `404 (Not Found)` status code at their `/.well-known/related-website-set.json` endpoint to demonstrate a deliberate desire to remove the set.
+	- Example for  `primary.com/.well-known/related-website-set.json`:
```

Comment:
I think there just isn't an example to share here, maybe this like should be removed @cfredric

Reply:
Yeah, looks like this line ends with `:` at HEAD too. I'm fine with removing this, though not sure if we want to do that in this PR since is purely formatting so far. It'd be nice to separate out substantive edits from formatting fixes.